content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -24,13 +24,22 @@ If you are upgrading from a version earlier than 4.0.0, these changes to index c
 
 You can configure indexes, unique constraints, and primary key constraints with the following attribute arguments:
 
-- The `length` argument is available **in MySQL only** on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. It allows Prisma to support indexes and constraints on `String` and `Bytes` types.
+- The `length` argument allows you to specify a maximum length for the subpart of the value to be indexed on `String` and `Bytes` types
 
-- The `sort` argument is available for all databases on the `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. Additionally, SQL Server also allows it on `@id` and `@@id`.
+  - Available on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes **in MySQL only**, in versions 3.5.0 and later
```

Comment:
Have you tried splitting this bullet into 3? Will cost vetrical space, but sperate the different information bits:
```suggestion
  - Available on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes 
  - **MySQL only**
  - Versions 3.5.0 and later
```
or similar

Reply:
Thanks, have tried this (see latest commit) and I think I prefer it